website/docs/general/events.md

```diff
@@ -0,0 +1,29 @@
+---
+id: events
+title: Events
+keywords:
+  - Events
+  - Conferences
+  - Workshops
+  - Meetups
+description: This page provides information about upcoming Apache APISIX's community events where you can track Meetups, Conferences and Workshops.
+---
+
+## Events
```

Comment:
Hi, bcoz we have the Page Title attribute, how about removing this one?

Reply:
Good point! Let's remove that.

Reply:
> Hi, bcoz we have the Page Title attribute, how about removing this one?

I removed the title of the event but added content navigation for different kind of events like:

## Contents

* [Conferences](#conferences)
* [Workshops](#workshops)
* [Meetups](#meetups)

Reply:
@juzhiyuan Now it looks better.

Reply:
Hi, Bobur, actually the right sidebar contains ToC 😂

Reply:
> Hi, Bobur, actually the right sidebar contains ToC 😂

Yes, now can we proceed with current solution?

Reply:
![image](https://user-images.githubusercontent.com/14247607/160079775-4d603c00-4d85-4846-a069-bce0148a1a34.png)